test/junit/scala/tools/nsc/interpreter/ScriptedTest.scala

```diff
@@ -1,10 +1,11 @@
 package scala.tools.nsc
 package interpreter
 
-import org.junit._, Assert._, runner.RunWith, runners.JUnit4
```

Comment:
this was a really ugly import, and I've no idea if the replacements are correct. they *look* correct on the surface, but I'd like to have a closer look at the code before saying so

Reply:
ugly in the sense of boilerplate.